airflow-local/templates/volume.yaml.tpl

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
This is a bit unfortunate, but there was just no way to get hostPath working by creating a DAG.